tensorflow/contrib/tensorrt/custom_plugin_examples/BUILD

```diff
@@ -0,0 +1,110 @@
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+load(
+    "//tensorflow:tensorflow.bzl",
+    "tf_custom_op_library",
+    "tf_cuda_library",
+    "tf_gen_op_libs",
+    "tf_gen_op_wrapper_py",
+    "tf_py_wrap_cc",
+    "tf_copts",
+)
+load(
+    "@local_config_tensorrt//:build_defs.bzl",
+    "if_tensorrt",
+)
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_py_library")
+load("//tensorflow:tensorflow.bzl", "tf_kernel_library")
+
+tf_kernel_library(
+    name = "_inc_op_plugin_kernel",
+    srcs = [
+        "inc_op_plugin.cc",
+    ],
+    hdrs = [
+    ],
+    gpu_srcs = [
+        "inc_op_kernel.cu.cc",
+        "inc_op_kernel.h",
+        "inc_op_plugin.h",
+    ],
+    deps = if_tensorrt([
+        "@local_config_tensorrt//:nv_infer",
+        "//tensorflow/contrib/tensorrt:trt_plugins",
```

Comment:
nit: This doesn't need to be inside if_tensorrt, please move it out here and below.